app/src/main/java/com/example/cinemapp/util/LanguageUtil.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.cinemapp.util
+
+import android.app.LocaleManager
+import android.content.Context
+import android.os.Build
+import android.os.LocaleList
+import androidx.appcompat.app.AppCompatDelegate
+import androidx.core.os.LocaleListCompat
+
+const val ENGLISH = "en-US"
+const val SERBIAN = "sr-RS"
+
+fun setupLanguage(language: String, context: Context) {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
```

Comment:
Avoid putting utility function in global scope, put it in a [singleton](https://medium.com/@ZahraHeydari/singleton-pattern-in-kotlin-b09380c53b14) `object LanguageUtil{}`

Reply:
Will be done in next PR.